packages/make-styles/src/runtime/resolveStyleRules.ts

```diff
@@ -113,6 +114,11 @@ export function resolveStyleRules(
         const combinedSupportQuery = generateCombinedQuery(support, property.slice(9).trim());
 
         resolveStyleRules(value, unstable_cssPriority, pseudo, media, combinedSupportQuery, result);
+      } else {
+        // static css (e.g. font-face)
+        const staticCSS = compileStaticCSS(property, value);
+        const staticCSSKey = HASH_PREFIX + hashString(staticCSS);
+        result[staticCSSKey] = [undefined, staticCSS /* no RTL support for static css */];
```

Comment:
Can we extend `MakeStyles` interface to include a definition for `@font-face`?

```ts
export interface MakeStyles extends Omit<CSSProperties, 'animationName'> {
  '@font-face': {
    fontFamily: string;
    src: string;

    fontFeatureSettings?: string;
    fontStretch?: string;
    fontStyle?: string;
    fontVariant: string;
    fontVariationSettings?: string;
    fontWeight?: number | string;

    unicodeRange?: string;
  };
}
```

[Taken from MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face#formal_syntax)